pkg/s3/bucket.go

```diff
@@ -124,3 +128,38 @@ func SetBucketLifecycle(s3Client *s3.S3, bucketName string) error {
 
 	return err
 }
+func CreateBucketTaggingInput(bucketname string, backUpLocation string) *s3.PutBucketTaggingInput {
+	putInput := &s3.PutBucketTaggingInput{
+		Bucket: aws.String(bucketname),
+		Tagging: &s3.Tagging{
+			TagSet: []*s3.Tag{
+				{
+					Key:   aws.String(bucketTagKey),
+					Value: aws.String(backUpLocation),
+				},
+			},
+		},
+	}
+	return putInput
+}
+
+func ClearBucketTags(s3Client *s3.S3, bucketName string) (err error) {
```

Comment:
Do we use this?

Reply:
ClearBucketTags is called on line 154. The spec called for correcting so they are deleted and reapplied according to spec each time through tag enforcement.

The return putInput is used on line 158 where it is passed to the s3 client for execution.